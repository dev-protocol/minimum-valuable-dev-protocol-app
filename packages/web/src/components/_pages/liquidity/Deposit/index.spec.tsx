import React from 'react'
import { render } from '@testing-library/react'
import { Deposit } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/_pages/liquidity/geyser/hooks.ts')
jest.mock('src/fixtures/_pages/liquidity/uniswap-pool/hooks.ts')

describe(`${Deposit.name}`, () => {
  test('Snapshot', () => {
    const component = render(<Deposit></Deposit>)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
