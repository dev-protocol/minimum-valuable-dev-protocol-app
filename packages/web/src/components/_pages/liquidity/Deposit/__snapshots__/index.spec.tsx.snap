// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deposit Snapshot 1`] = `
<body>
  <div>
    <form
<<<<<<< HEAD
      class="ant-form ant-form-vertical pju26w-0 bSBNGA"
=======
      class="ant-form ant-form-vertical pju26w-0 lnGzgd"
>>>>>>> upstream/main
    >
      <div
        class="ant-statistic"
      >
        <div
          class="ant-statistic-title"
        >
          Program duration
        </div>
        <div
          class="ant-statistic-content"
        >
          <span
            class="ant-statistic-content-value"
          >
            0 day(s) 00:00:00 left
          </span>
        </div>
      </div>
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="sc-69st3h-0 iPdCKq"
              >
                <label
                  for="amount"
                >
                  Your amount
                </label>
                <span
                  class="ant-input-affix-wrapper ant-input-affix-wrapper-lg sicgyx-0 cyULWd"
                >
                  <input
                    class="ant-input ant-input-lg"
                    id="amount"
                    type="number"
                    value=""
                  />
                  <span
                    class="ant-input-suffix"
                  >
                    <span
                      class="li6qma-0 YWKly"
                    >
                      ETHDEV-V2
                    </span>
                    <button
                      class="ant-btn ant-btn-primary ant-btn-sm zinbpr-0 cYMWzK"
                      type="button"
                    >
                      <span>
                        Max
                      </span>
                    </button>
                  </span>
                </span>
                <a
                  href="https://app.uniswap.org/#/add/0x5caf454ba92e6f2c929df14667ee360ed9fd5b26/ETH"
                  rel="noreferrer"
                  target="_blank"
                >
                  Provide liquidity on Uniswap to get ETHDEV-V2 â†—
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="sc-69st3h-0 iPdCKq"
              >
                <label
                  for="estimated"
                >
                  Your estimated reward
                </label>
                <input
                  class="ant-input ant-input-lg ant-input-disabled sicgyx-0 cyULWd"
                  disabled=""
                  id="estimated"
                  type="text"
                  value="0 / month"
                />
                <small>
                  *Estimated rewards assume you have achieved the maximum reward multiplier.
                </small>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="ant-steps ant-steps-vertical ant-steps-dot"
              >
                <div
                  class="ant-steps-item ant-steps-item-process ant-steps-item-active"
                >
                  <div
                    class="ant-steps-item-container"
                  >
                    <div
                      class="ant-steps-item-tail"
                    />
                    <div
                      class="ant-steps-item-icon"
                    >
                      <span
                        class="ant-steps-icon"
                      >
                        <span
                          class="ant-steps-icon-dot"
                        />
                      </span>
                    </div>
                    <div
                      class="ant-steps-item-content"
                    >
                      <div
                        class="ant-steps-item-title"
                      >
                        Approve
                      </div>
                      <div
                        class="ant-steps-item-description"
                      >
                        <p>
                          Please approve the token transfer to deposit ETHDEV-V2.
                        </p>
                        <button
                          class="ant-btn ant-btn-primary ant-btn-lg"
                          type="button"
                        >
                          <span>
                            Approve
                          </span>
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="ant-steps-item ant-steps-item-wait"
                >
                  <div
                    class="ant-steps-item-container"
                  >
                    <div
                      class="ant-steps-item-tail"
                    />
                    <div
                      class="ant-steps-item-icon"
                    >
                      <span
                        class="ant-steps-icon"
                      >
                        <span
                          class="ant-steps-icon-dot"
                        />
                      </span>
                    </div>
                    <div
                      class="ant-steps-item-content"
                    >
                      <div
                        class="ant-steps-item-title"
                      >
                        Deposit
                      </div>
                      <div
                        class="ant-steps-item-description"
                      >
                        <p>
                          Deposit ETHDEV-V2.
                        </p>
                        <button
                          class="ant-btn ant-btn-primary ant-btn-lg"
                          disabled=""
                          type="button"
                        >
                          <span>
                            Deposit
                          </span>
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </form>
  </div>
</body>
`;
